src/components/Popover/FilterPopover.css

```diff
@@ -0,0 +1,16 @@
+.date-picker-filter {
+    max-width: 200px;
+}
+
+.link-button {
+    /* https://stackoverflow.com/questions/1367409/how-to-make-button-look-like-a-link */
+    background: none!important;
+    border: none;
+    padding: 0!important;
+    /*optional*/
+    font-family: arial, sans-serif;
+    /*input has OS specific font-family*/
+    color: #069;
+    text-decoration: underline;
+    cursor: pointer;
+}
```

Comment:
I mention this elsewhere, but should we just use <Button variant="link">Link</Button> ?